shared/src/test/diff/typegen/TypegenTypedefs.mls

```diff
@@ -160,3 +160,38 @@ class E: { x: "hello"; y: int } & A1 & B2 & D3
 //│     constructor(fields: {y: int, x: "hello" & ((arg: int) => int)})
 //│ }
 //│ // end ts
+
+:ts
+type VarArg[T] = (T, T)
+class Program[T]
+  method Run: VarArg[T] -> number
+//│ Defined type alias VarArg
+//│ Defined class Program
+//│ Declared Program.Run: Program['T] -> ('T, 'T,) -> number
+//│ // start ts
+//│ export type VarArg = readonly [T, T]
+//│ export declare class Program<T> {
+//│     constructor(fields: {})
+//│     Run(arg: readonly [T, T]): number
+//│ }
+//│ // end ts
+
+:ts
+// FIXME
```

Comment:
Is there actually anything to fix here?

Reply:
It's more like a todo. I was hoping to add an example where AppliedType hides the tuple argument. But couldn't find one.

I should remote this and such an example later?

Reply:
You can leave the test (it's always useful to have it), but  remove the FIXME.

The reason you can't reproduce the issue is that the current type simplifier automatically expands all type aliases. But this will probably change in a future version.

Reply:
Replaced fixme with :e